applications/FluidDynamicsApplication/python_scripts/compute_drag_process.py

```diff
@@ -102,7 +102,8 @@ def ExecuteFinalizeSolutionStep(self):
                     self._PrintToScreen(result_msg)
 
                 if (self.write_drag_output_file):
-                    self.output_file.write(str(current_time)+" "+format(drag_force[0],self.format)+" "+format(drag_force[1],self.format)+" "+format(drag_force[2],self.format)+"\n")
+                    self.output_file.write(format(current_time, self.format)+" "+format(drag_force[0],self.format)+" "+format(drag_force[1],self.format)+" "+format(drag_force[2],self.format)+"\n")
+                    self.output_file.flush()
```

Comment:
After a long discussion with @msandre and @jcotela it was decided to NOT do this due to performance reasons

If you want it to always flush look into the class this process uses and (TimeBasedAsciiWriter) and decrease the buffer size to 1 (this value comes from python)

Reply:
I see the concern about the flush function. In such case .. i would propose that we force this value of 1 ... I am saying considering that in this specific case its just 4 doubles we are writing here in this process ... And a lot of them can stay in the buffer before being written to file. This is a bit in convenient especially when we are giving a live demo .. (which happened to me today)

Reply:
The reason I am normally against forced flush is that disk access is an important bottleneck in clusters, so one should try to minimize it, especially in MPI. I know it is not your case here, but still this kind of code tends to be reused (or inherited) for the trilinos version of the thing.

I would be in favour of not putting the flush but instead setting the default of the AsciiWritter to 1 in OpenMP classes only (if it can be done easily or through json).

Reply:
Ok then i will go try to do it the second way ... !

Reply:
I really like the idea of setting it to on in omp!
@adityaghantasala two comments on how to do it (bcs the DragProcess does also work in MPI): 
- can you please use the same mechanism as I used [here](https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/python_scripts/save_restart_process.py#L42-L48), then the user does not have to set this setting. This way you also don't have to touch the AcsiiWriter.
- Can you please also do the same in the `PointsOutputProcess`, the `MultiplePointsOutputProcess` and the `LineOutputProcess`? @mpentek and I spent really a lot of work having all the processes using the AsciiWriter consistent and working well, it would be a shame to deviate so soon already

Reply:
Sure I will ... In any case i was not thinking of not using the AsciiWriter process ... :-)

Reply:
Cool thx

Reply:
If one has to put the check of mpi or openmp in all PointsOutputProcess, LineOutputProcess, MultiplePointsOutputProcess and ComputeDragProcess ... it means mostly the same check (same lines of code) done each time ... In that case I think putting such a check in TimeBasedAsciiWriter is much easier ... ! 
I was thinking like : 
If MPI and nothing is provided :  use the default (-1) 

If OpenMPI and nothing is provided : use 1

If something is provided : always use the provided one.

Reply:
Ah now I see
The TimeBasedAsciiWriter has the ModelPart, then it is easy
For some reason I though it does not have the ModelPart

Ok then it is easy, please go ahead, sorry for the noise

Reply:
Done !